src/features/appearance/index.ts

```diff
@@ -3,6 +3,7 @@ import { reaction } from 'mobx';
 import { DEFAULT_APP_SETTINGS, iconSizeBias } from '../../config';
 
 const STYLE_ELEMENT_ID = 'custom-appearance-style';
+const isWindows = process.platform === 'win32';
```

Comment:
There's a constant already exported for `isWindows` from `environment.ts` - please use that

Reply:
Fixed in latest commit